robot/util/rosRoverStart/local_rover.launch

```diff
@@ -2,4 +2,17 @@
     <node pkg="mux_selector" type="mux_select_server.py" name="mux_select_server" output="screen" args="local"/>
     <node pkg="web_video_server" type="web_video_server" name="web_video_server" output="screen"/>    
     <node pkg="task_handler" type="task_handler_server.py" name="task_handler_server" output="screen" args="local"/>
+    <node pkg="mcu_control" type="MockPdsNode.py" name="mock_pds_node" output="screen"/>
+    <param name="PDS_mock_rise_fall_rate" value="stable"/>
```

Comment:
what is this

Reply:
why is it "stable"? Shouldnt it be a float?

Reply:
Im reusing the 3 possible states for the rise/fall rate (rise = 0.1, stable = 0.04, and 0.01). Unless you want the user to be able to manually enter a value of their liking.

Reply:
when I tested to "rise" temp1 it would go to 100 instantaneously

Reply:
Fixed this